### ISSUE TITLE
Feedback about GLU

### ISSUE THREAD

**Issue by vhrique** [pytorch/pytorch#167240]
Is the Figure correct in this page? Isn't the plot actually related to Gaussian Error Linear Unit (GeLU) or Sigmoid Linear Unit (SiLU) instead of Gated Linear Unit (GLU)?

There is the following issue on this page: https://docs.pytorch.org/docs/stable/generated/torch.nn.GLU.html

cc @svekars @sekyondaMeta @AlannaBurke @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki
